docs/internals/deprecation.rst

```diff
@@ -44,10 +44,21 @@ Compat Task Modules
 
     into:
 
+    .. code-block:: python
+
+        from celery import shared_task
+
+    -- and:
     .. code-block:: python
 
         from celery import task
```

Comment:
from celery import task should be Task

Reply:
no, shared_task is not a decorator? it is different from shared_task